src/main/plugins/org.dita.pdf2/xsl/fo/topic.xsl

```diff
@@ -58,7 +58,7 @@ See the accompanying LICENSE file for applicable license.
         </xsl:variable>
         <xsl:choose>
             <!--  Disable chapter title processing when mini TOC is created -->
-            <xsl:when test="(topicType = 'topicChapter') or (topicType = 'topicAppendix')" />
+            <xsl:when test="($topicType = 'topicChapter') or ($topicType = 'topicAppendix')" />
```

Comment:
This looks like it fixes a very old bug, and will change the layout of the mini-TOC in PDF; I generally turned off the mini-TOC but I wonder if this will give unexpected results. (That is, maybe if this had worked in the first place, the behavior would be wrong, and we haven't noticed because the code bug masked that issue.)

Reply:
@robander - I also wondered about behavior change when filing the PR, but I could not sensitize this statement. For example, if I add the following debug statement:

```
<xsl:message expand-text="yes">$topicType = '{$topicType}'</xsl:message>
```

then transform the following `<bookmap>` testcase with `<part>`/`<chapter>`/`<appendices>`/`<appendix>` structure:

[4093-bookmap.zip](https://github.com/dita-ot/dita-ot/files/11342341/4093-bookmap.zip)

I only get `topicSimple` values:

```
transform.topic2fo.main:
...
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
 [pipeline] $topicType = 'topicSimple'
```

Maybe there is another bug with `topicChapter` and `topicAppendix` values that has been hiding this bug? We don't use the `pdf2` transform so this code is unfamiliar to me.

Reply:
@robander - I think the code to check `$topicType` here might be orphaned code. In `plugins/org.dita.pdf2/xsl/fo/commons.xsl`, there are separate templates for the bookmap structures, and these process the `<title>` elements themselves:

```
$ rg 'xsl:template name=.*processTopic\w' build/tmp/dist/plugins/org.dita.pdf2/xsl/fo
build/tmp/dist/plugins/org.dita.pdf2/xsl/fo/preface.xsl
41:     <xsl:template name="processTopicPreface">

build/tmp/dist/plugins/org.dita.pdf2/xsl/fo/commons.xsl
231:    <xsl:template name="processTopicChapter">
306:    <xsl:template name="processTopicAppendix">
381:  <xsl:template name="processTopicAppendices">
463:    <xsl:template name="processTopicPart">
543:    <xsl:template name="processTopicNotices">
```

Perhaps in earlier days, the titles of all topic types went through this `topic/title` template, but now that's no longer true? If so, we could remove this check completely and redirect the `topic/title` template processing right into `mode="processTopicTitle"`.